### DISCUSSION TITLE
Water level constituents comparisons

### DISCUSSION THREAD

**Discussion by jaadhi** [DHI/modelskill#14] [Ideas]
Table comparing measured and modelled water level constituents

 
![image](https://user-images.githubusercontent.com/47220706/107642359-10936280-6c75-11eb-9188-1a99c411693d.png)

**Comment by jsmariegaard** [comment]
We definitely need some way of analysing and comparing tides. I like this table. For some of the values (like amplitude) a bar chart could also be useful.
